src/pages/auth/login.tsx

```diff
@@ -29,22 +24,17 @@ const LoginPage = () => {
     setEmail(e.target.value);
   };
 
-  const handleSubmit = (e: React.ChangeEvent<HTMLInputElement>) => {
+  const handleSubmit = async (e: React.ChangeEvent<HTMLInputElement>) => {
     e.preventDefault();
 
-    if (email === testAcount.email) {
-      setEmailInvalid(false);
-    } else {
-      setEmailInvalid(true);
-    }
-    if (password === testAcount.password || password.length <= 5) {
-      setPasswordInvalid(false);
-    } else {
-      setPasswordInvalid(true);
-    }
-
-    if (email === testAcount.email && password === testAcount.password) {
-      alert(`Login maded with sucess. Welcome ${email}!!`);
+    try {
+      const response = await UserService.login({ email, password });
+      if (response.message === 'Incorrect email or password') {
+        setEmailInvalid(true);
+        setPasswordInvalid(true);
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
Let's remove this console.log and handle the catch:
- Create a shared component that will show a toast[danger] with "Oops. Something went wrong"
- It will appear for all the generic errors in the application like catchs